docs/spark.rst

```diff
@@ -96,7 +96,33 @@ logging (for Tensorboard) using the Estimator ``Store`` abstraction.  Stores are
 artifacts including intermediate representations of the training data.  Horovod natively supports stores for HDFS
 and local filesystems.
 
-`Petastorm <https://github.com/uber/petastorm/blob/master/petastorm/pytorch.py#L258>` based data loader is used by default, but user can define a custom data loader by override the `BaseDataLoader` interface. A async data loader mixin can also added on top of the data loader.
+`Petastorm <https://github.com/uber/petastorm/blob/master/petastorm/pytorch.py#L259>`__ based data loader is used by default,
+but user can define a custom data loader by overriding the `BaseDataLoader` interface. An async data loader mixin can also
+be added on top of the data loader.  Additionally, the KerasEstimator supports a DataModule argument, similar
+to the Lightning DataModule, which abstracts the data loading and allows for alternative implementations.  For example,
+the NVTabularDataModule integrates the `KerasSequenceLoader <https://github.com/NVIDIA-Merlin/NVTabular/blob/main/nvtabular/loader/tensorflow.py>`__
+from NVTabular to enable GPU-accelerated data loading.  Note, however, due to the complexity of installation, NVTabular
+recommends the use of a `conda` environment or a pre-built docker image.  For users who want to build their own docker images,
+there is an `example Dockerfile <../docker/horovod-nvtabular/Dockerfile>`__ for building Horovod with NVTabular support.
```

Comment:
This documentation will be hosted separate from the sources, so this link will be broken:
https://horovod--3603.org.readthedocs.build/en/3603/spark_include.html#horovod-spark-estimators

Please link to https://github.com/horovod/horovod/blob/master/docker/horovod-nvtabular/Dockerfile. That link will become live once this is merged into master.